### PR TITLE
Release notes for 3.1.0 as well as some updates to other 3.1 docs content

### PR DESCRIPTION
**Pull request by zhill**

### CODE REVIEW

File: content/docs/installation/anchorectl/_index.md [hunk 1]
```diff
@@ -8,58 +8,49 @@ weight: 3
 
 In this section you will learn how to install and configure AnchoreCTL, the Anchore Enterprise CLI.
 It currently shares some functionality with anchore-cli is designed specifically for use with Anchore Enterprise. 
-AnchoreCTL is published as a Cobra-CLI Go Binary that can be installed from a public S3 bucket. 
+AnchoreCTL is published as simple binary that can be installed by downloading it or using provided packages for installation in different platforms.
```
Comment (dspalmer99) [inline content/docs/installation/anchorectl/_index.md:11]: a simple binary

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This binary is installed into the Enterprise Docker Image and can be configured
 
 ## Agent Mode
 The most common way to track inventory is to install [`kai`](https://github.com/anchore/kai). as an agent in your cluster. To do this you will need to configure credentials
```
Comment (dspalmer99) [inline content/docs/overview/concepts/images/inventory/_index.md:13]: There's an extra period at the end of [`kai`](https://github.com/anchore/kai).

File: content/docs/releasenotes/3.1.0/_index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+## Enterprise Service Changes
+
+This release contains a database schema update to version 0.0.8 for the enterprise schema and 0.0.15 for the engine schema.
+The upgrade process will modify the db schema and update some tables in the reporting service for any existing runtime
+inventory records. Unless you have a very large number of inventory records, the upgrade should complete in seconds to minutes depending
+on your database size.
+
+### Owned Package Filtering Control
+A new configuration option: services.analyzer.enable_owned_package_filtering: <bool> is now available in the analyzer service configuration. 
+By default, the analyzer will filter packages that are determined by at analysis time to be "owned" by a parent package when that package
```
Comment (dspalmer99) [inline content/docs/releasenotes/3.1.0/_index.md:50]: Small typo, extra "by" in "that are determined by at analysis"

File: content/docs/releasenotes/3.1.0/_index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+### Fixed
+- Adds missing fields and fixes errors in the swagger spec for the API
+- Restores file package verification data ingress during image load to fix a regression
+- Malware policy gate can fail causing policy eval error when malware not enabled and other rules precede malware rule in a policy
+- JSON serialization error in internal policy engine user image listing API
+- "package_cpe23" field missing in vulnerabilities
+- Ensure python38 used in the Dockerfile build, and set tox tests to only run py38
+- User to not be able to delete some notification configurations when they should be able based on RBAC role
+
```
Comment (dspalmer99) [inline content/docs/releasenotes/3.1.0/_index.md:71]: Extra line break here

File: content/docs/releasenotes/3.1.0/_index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+### Improved
+- Performance of GET operations between services improved by better streaming memory management for large payload transfers
+- Use UBI 8.4 as base image in Docker build
+- Updates skopeo version used to 1.2.1, allowing removal of the 'lookuptag' field in the POST /repositories call for 
+  watching repositories that do not have a latest tag
+- RedHat packages for an Out-of-Support distro release version now indicated as being vulnerable if a newer distro release version is supported and indicated as affected for the package.
+Additional minor bug fixes and enhancements
+
```
Comment (dspalmer99) [inline content/docs/releasenotes/3.1.0/_index.md:80]: Extra line break here

File: content/docs/releasenotes/3.1.0/v2_vulnerability_scanning.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+     1. `vulnerability_data_unavailable`
+     1. `stale_feed_data policy`
+4. Windows container scanning is not yet supported. Support will be added in the next feature release.
+5. Proprietary vulnerability feeds are not yet supported in this scanner. Support will be added in the next feature release.
+
+### Feed Syncs
+
+The new vulnerability provider, when configured for use, will sync feeds from the on-premises feed service but will expect
+a different format for the feeds data. This format is consistent with the Grype DB, but is built using the data from your
+deployed feed service. In future releases, the Microsoft MSRC and VulnDB data soures will be supported as well
```
Comment (dspalmer99) [inline content/docs/releasenotes/3.1.0/v2_vulnerability_scanning.md:28]: soures -> sources

File: content/docs/using/ui_usage/kubernetes/_index.md [hunk 7]
```diff
@@ -3,23 +3,23 @@ title: "Kubernetes Inventory"
...
+In addition to navigating your runtime inventory by clusters and namespaces, users can opt to view the images or vulnerabilities across. This is a great way to identify vulnerabilities across your runtime and asses their impact.
 
-### Assesing impact
-Another important aspect of the Kubernetes Inventory UI is the ability to asses how a vulnerability in a container images impacts your environment. For every container when you see a note about it usage being seen in particular cluster and X more... you will be able to mouse over the link for a detailed list of where else that container image is being used. This is quick way to determine the "bast-radius" of a vulnerabilty. 
+### Assessing impact
+Another important aspect of the Kubernetes Inventory UI is the ability to assess how a vulnerability in a container images impacts your environment. For every container when you see a note about it usage being seen in particular cluster and X more... you will be able to mouse over the link for a detailed list of where else that container image is being used. This is fast way to determine the "bast-radius" of a vulnerability.
```
Comment (dspalmer99) [inline content/docs/using/ui_usage/kubernetes/_index.md:19]: bast-radius -> blast-radius

File: content/docs/releasenotes/3.1.0/_index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+### Known Issues/Errata
+Note: the policy engine feed sync configuration is now in the policy engine service configuration as part of the provider 
+configuration. The provided helm charts, docker-compose.yaml and default configurations handle this change automatically.
+
+### Deprecations
+The `affected_package_version` query parameter in GET /query/vulnerabilities is not supported in the V2 scanner (aka Grype mode) 
+and has known correctness issues in the legacy mode. It is deprecated and will be removed in a future release.
+
+
+## Enterprise UI Changes
```
Comment (dspalmer99) [inline content/docs/releasenotes/3.1.0/_index.md:91]: Are these sections still forthcoming? Happy to take another look afterwards.

File: content/docs/using/ui_usage/kubernetes/_index.md [hunk 9]
```diff
@@ -3,23 +3,23 @@ title: "Kubernetes Inventory"
...
 
 ## Data Delays
 Due to the processing required to generate the data used by the Kubernetes Inventory UI, the results displayed may not be fully up to date. The ovreall delay depends on the configuration of how often inventory data is collected, and how frequently your reporting data is refreshed. This is similar to delays present on the dashboard. 
 
 ## Policy and Account Considerations 
-The Kubernetes Inventory is only availale for an Account's default policy. You may want to consider setting up an account specifically for tracking your Kubernetes Inventory and enforcing a policy. 
\ No newline at end of file
+The Kubernetes Inventory is only available for an Account's default policy. You may want to consider setting up an account specifically for tracking your Kubernetes Inventory and enforcing a policy.
```
Comment (dspalmer99) [inline content/docs/using/ui_usage/kubernetes/_index.md:25]: There's a capitalization mismatch between the two uses of "account" in this paragraph.

**Review by dspalmer99 — APPROVED** [review]
A few typos, but overall this is looking good.

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 10]
```diff
@@ -46,15 +46,15 @@ Note: the Anchore API Password can be provided via a kubernetes secret, or injec
...
 
 ### Usage
-To quickly verify that you are tracking Kubernetes Inventory you can quickly access inventory results from the `anchorectl` with the  command `anchorectl inventory list`. The UI for the Kubernetes allows operators to visually navigate vulnerability resultes, source where vulnerabilties come from and apply their default policy to thier runtime. You'll be able to subscribe to changes for your clusters, quicky asses vulnerabilities and more.
+To quickly verify that you are tracking Kubernetes Inventory you can quickly access inventory results from the `anchorectl` with the  command `anchorectl inventory list`. The UI for the Kubernetes allows operators to visually navigate vulnerability resultes, source where vulnerabilities come from and apply their default policy to their runtime. You'll be able to subscribe to changes for your clusters, quickly asses vulnerabilities and more.
```
Comment (Toure) [inline content/docs/overview/concepts/images/inventory/_index.md:55]: The second use of quickly could be removed as the initial use of quickly sets the intent for the sentence.

**Review by Toure — CHANGES_REQUESTED** [review]

**Comment by dspalmer99** [comment]
Tested the changed curls, all worked correctly.

File: content/docs/releasenotes/3.1.0/_index.md (hunk 8 above)
Comment (zhill) [reply content/docs/releasenotes/3.1.0/_index.md:89]: Yes these will be updated by @holt with UI updates before publication

**Comment by zhill** [comment]
Thanks for the reviews!

**Pull request merged**